### PR TITLE
[MDB Ignore] Converts A Bunch Of Varedited Windows To Spawners

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Does what it says on the tin. Contains the viral spread of varedited (And in some cases, stacked) windows (Except on Crashed Clown Ship, I didn't have the holy status to fix that mess) - I didn't touch the basic syndicate infiltrator because #65880 is still open, and that *did* have the fixes this PR has there. I will forward the fix should that be closed.

I've gone *painstakingly* - emphasis on pain - through the repo to fix as much as I can. Here's the original checklist from when this PR was a draft, in all it's completed glory:

- [x] Shuttles
- [x] Generic Map Templates
- [x] Away Missions
- [x] CTF Maps
- [x] Stations
- [x] CentCom.dmm
- [x] Lavaland.dmm (Literally just fixing what's under one window but whatever)
- [x] Anywhere Ruins (Literally just the golem ship but shut up)
- [x] Lavaland Ruins
- [x] Space Ruins (Minus Crashed Clown Ship, because.. colors...)
- [x] Icemoon Ruins

...Along with fixing a few issues I sighted that wouldn't distract from the goal of this PR. Nothing *too* out there. These comprise the only player-facing changes of this PR.

These include:
A bunch of /area/ weirdness in icemoon_plasma_facility and the space hotel. Iunno who spilled ink on the map, but they sure did.
Mafia_Lavaland would load space. Funny as that is, that's not good yo!!!!!
The stools being rotated the wrong way in the luxury bar capsule. This was vital, shut up. Shut up right now.

## Why It's Good For The Game
VAREDIT? MORE LIKE RIP AND TEAR-EDIT

## Changelog

:cl:
fix: The lavaland mafia arena will no longer load space, the Luxury Bar Survival Capsule's stools are rotated the right way, and the way APCs work in the icemoon plasma facility ruin no longer is.. BAD.
/:cl:

### CODE REVIEW

**Comment by dragomagol** [comment]
I hope you're not treading the same dark path I did by doing this all manually, if you make a file for this in `tools/UpdatePaths` it'll replace them all automatically on all maps.

**Comment by unit0016** [comment]
> I hope you're not treading the same dark path I did by doing this all manually

💀

**Comment by Fikou** [comment]
why

**Comment by unit0016** [comment]
> why

Killing off map varedits is good actually

**Comment by Fikou** [comment]
> > why
> 
> Killing off map varedits is good actually

why

**Comment by NamelessFairy** [comment]
> > why
> 
> Killing off map varedits is good actually

From my experience with strongDMM it definitely favors directional edits over new subtypes. It allows for a directional window for example to be dropper tooled and all its alternate directions are available within a separate panel rather than needing to select a different path. So personally I'd prefer dir edits to remain and would like to echo fikous question of "why"

**Comment by unit0016** [comment]
> why

Because they **stack.** And cause an ungodly amount of bloat.

Sure, one little varedit to a floor type or a wall or what have you seems harmless, an extra four odd lines.
That is, until you realize that for every time you use that varedited object, it's redefined for every minuet difference, from /area/ to a chair to just a decal on top of it.

Look at the amount of lines removed versus kept in this PR, and the absurd difference that's stacked just from the files I've already dealt with.

But that's just for the panes - what about fulltile windows? Well, there's a nifty difference with the spawner that's semi-important.  See, spawners have a hole in the middle to let you peek under what's under them. Be it a misplaced cable, pipe, floor, what have you. That and it saves one extra line where the grille was also placed.

tl;dr, it's good practice to not do varedits in the map file itself.

**Comment by unit0016** [comment]
Turbine rework got merged, time to go back in and make sure that didn't break anything after I run mapmerge
